curriculum/challenges/english/01-responsive-web-design/css-nutrition-label/step-028.md

```diff
@@ -0,0 +1,99 @@
+---
+id: 615f50473cc0196c6dd3892a
+title: Step 28
+challengeType: 0
+dashedName: step-28
+---
+
+# --description--
+
+You may notice there is still a small border at the bottom of your `.lg` element. To reset this, give your `.lg, .md` selector a `border` property set to `0`.
```

Comment:
This is the most subtle change I have seen. Just saying 🤷‍♂️

Reply:
Subtlety is an art 🙃